src/utils/handlers/general/searchTrack.ts

```diff
@@ -115,6 +117,13 @@ export async function searchTrack(
                                 )
                             );
 
+                            if (songIndex) {
+                                const temp = parseInt(songIndex) < 101 ?
+                                    tracks.splice(parseInt(songIndex) - 1, 1)[0]
+                                    : null;
+                                if (temp) tracks.unshift(temp);
```

Comment:
There is warning `Blocks are nested too deeply (6). Maximum allowed is 5`

Reply:
had to declare temp outside so it doesn't have to nest so far